client/components/layout/Block/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react'
+
+import Base from '../internal/Base'
+
+const propTypes = {
+  align: PropTypes.oneOf(['left', 'right']),
+  children: PropTypes.node,
+  className: PropTypes.string,
+  shrink: PropTypes.bool,
+  vertical: PropTypes.bool
+}
```

Comment:
None of these props are used in this class, so don't need to be validated here.  I can see keeping them as documentation, though.